components/log/Kconfig

```diff
@@ -44,5 +44,17 @@ menu "Log output"
 
             In order to view these, your terminal program must support ANSI color codes.
 
+    config LOG_SYSTEM_TIME
```

Comment:
To make it easier for users to see the possibilities here, could you please make this a `choice` config item "Log timestamps", with two choices "System time" & "Milliseconds since boot"?